io.grpc.java/pom.xml

```diff
@@ -0,0 +1,161 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Why are we still using maven for this ?

Reply:
All of the bundles in wrapping use this - what do you recommend it should be?